Automattic/woocommerce-payments
# Migrate login_links, capital_links, tos_agreement to use the core Request class

Fixes #
A part of https://github.com/Automattic/woocommerce-payments-meta/issues/87

**ℹ️ NOTICE** 

- This PR is based on https://github.com/Automattic/woocommerce-payments/pull/5879's PR branch. Hence, 5879 should be merged first though this PR can be reviewed at any time.

#### Changes proposed in this Pull Request

Migrate login_links, capital_links, tos_agreement to use the core Request class: 

- Update tests.
- Add new tests.
- Add MD docs.









#### Testing instructions

- Check all PHPUnit tests (update and new tests), and ensure they look good. 





*

-------------------


**Post merge**
